common/lib/xmodule/xmodule/tests/test_poll.py

```diff
@@ -30,3 +33,29 @@ def test_good_ajax_request(self):
         self.assertEqual(total, 2)
         self.assertDictEqual(callback, {'objectName': 'Conditional'})
         self.assertEqual(self.xmodule.poll_answer, 'No')
+
+    def test_poll_export_with_unescaped_characters_xml(self):
+        """
+        Make sure that poll_module will export fine if its xml contains
+        unescaped_characters.
+        """
+        module_system = DummySystem(load_error_modules=True)
+        id_generator = Mock()
+        id_generator.target_course_id = self.xmodule.course_id
+        sample_poll_xml = '''
+        <poll_question display_name="Poll Question">
+            <p>How old are you?</p>
+            <answer id="less18">18</answer>
+        </poll_question>
+        '''
+
+        output = PollDescriptor.from_xml(sample_poll_xml, module_system, id_generator)
+        # Update the answer with invalid character.
+        invalid_characters_poll_answer = output.answers[0]
+        invalid_characters_poll_answer['text'] = '< 18'  # invalid greater-than character.
+        output.answers[0] = invalid_characters_poll_answer
+        output.save()
+
+        xml = output.definition_to_xml(None)
+        child_texts = xml.xpath('//text()')  # extract texts of all children.
+        self.assertEqual(child_texts[3], '< 18')
```

Comment:
Why `child_texts[3]` ? Also, this is failing the python test. Can we not use `child_texts.answer` or something like that.